netsocadmin/routes/login.py

```diff
@@ -39,6 +40,7 @@ def dispatch_request(self) -> str:
         # Set the session info to reflect that the user is logged in and redirect back to /
         flask.session[config.LOGGED_IN_KEY] = True
         flask.session["username"] = flask.request.form["username"]
+        self.logger.debug("successfully logged in")
```

Comment:
Maybe change this to "user successfully logged in"
Even better would be if we could log *who* logged in.
Same for "incorrect password" and "logged out", knowing for who would be good